### PR TITLE
[INLONG-5269][Manager] Workflow supports rapid scaling of MQ resources

### LINKED ISSUE #5269: [Feature][Manager] Workflow supports rapid scaling of MQ resources
### Description

Workflow supports rapid scaling of MQ resources.

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Prepare a Pull Request

- Fixes #5269

### Motivation

Workflow supports rapid scaling of MQ resources.

### Modifications

1. Change the NORMAL_MODE to STANDARD_MODE for the inlong group.
2. Move the listener classes into the listener package.
3. Use factory design pattern, abstract the operation of MessageQueue, and remove XxxPulsarTaskListener and XxxTubeTaskListener.

### Verifying this change

- [X] This change is already covered by existing tests, such as:
    WorkflowServiceImplTest

### Documentation

  - Does this pull request introduce a new feature? (yes)
  - If yes, how is the feature documented? (JavaDocs)

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperatorFactory.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    /**
+     * Get a message queue resource operator instance via the given mqType
+     */
+    public QueueResourceOperator getInstance(MQType mqType) {
+        Optional<QueueResourceOperator> instance = operatorList.stream()
+                .filter(inst -> inst.accept(mqType))
+                .findFirst();
+        if (!instance.isPresent()) {
+            throw new BusinessException(ErrorCodeEnum.MQ_TYPE_NOT_SUPPORT,
+                    String.format(ErrorCodeEnum.MQ_TYPE_NOT_SUPPORT.getMessage(), mqType));
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperatorFactory.java:47]: Could be combined like this
```
return operatorList.stream()
                .filter(inst -> inst.accept(mqType))
                .findFirst()
                .orElseThrow(() -> new BusinessException(ErrorCodeEnum.MQ_TYPE_NOT_SUPPORT,
                        String.format(ErrorCodeEnum.MQ_TYPE_NOT_SUPPORT.getMessage(), mqType)));
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+    @Override
+    public boolean accept(MQType mqType) {
+        return MQType.PULSAR == mqType || MQType.TDMQ_PULSAR == mqType;
+    }
+
+    @Override
+    public void createQueueForGroup(InlongGroupInfo groupInfo, String operator) {
+        Preconditions.checkNotNull(groupInfo, "inlong group info cannot be null");
+        Preconditions.checkNotNull(operator, "operator cannot be null");
+        String groupId = groupInfo.getInlongGroupId();
```
Comment (leosanqing) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java:71]: I think a blank line between the verification code and the business code will be more readable。

The same below

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperatorFactory.java (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperatorFactory.java]: done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java (hunk 2 above)
Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java:72]: Okay, already added a blank line after the verification codes.

**Review by kipshi — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by leosanqing — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/mysql/MySQLResourceOperator.java [hunk 3]
```diff
@@ -1,137 +1,137 @@
...
+        MySQLTableInfo tableInfo = MySQLSinkDTO.getTableInfo(mySQLSink, columnList);
+        String url = mySQLSink.getJdbcUrl();
+        String user = mySQLSink.getUsername();
+        String password = mySQLSink.getPassword();
+
+        String dbName = tableInfo.getDbName();
+        String tableName = tableInfo.getTableName();
+        Connection conn = null;
+        try {
+            conn = MySQLJdbcUtils.getConnection(url, user, password);
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/mysql/MySQLResourceOperator.java:105]: It is better to replace the finally block with the try-with-resource statement.

**Review by haibo-duan — APPROVED** [review]

**Pull request merged**